### DISCUSSION TITLE
Affichage tableau de bord  en direct

### DISCUSSION THREAD

**Discussion by Baptiste-ALL** [witsa/synapps#756] [Q&A] [answered]
Bonjour,
J'ai plusieurs tableau de bord sur le Wit et j'aimerais avoir des accès direct a partir de Synapps, j'ai essayé de rentré l'adresse direct " easy/dashboard?Nodld=146539&" mais cela ne marche pas il reste sur le premier en haut de la liste. et justement et 'il aussi possible de supprimé la liste juste l'affichage du tableau?

D'avance merci

**Comment by WITNicolas** [comment, marked as answer]
Bonjour,

Ouvrez les tableau de bord en pleine page, vous aurez alors tous les numéros des NOD
![image](https://github.com/user-attachments/assets/bbcc319d-47e5-487d-bd87-6a30918d0734)

/easy/Dashboard-1?NodId=186303&

Si vous ne voulez pas la barre de navigation à droite, il faudra jouer sur les MARGES pour décaler de maniére négative sur la gauche pour la masquer. A priori il n'y a pas d'instruction pour masquer la liste

Voici l'acteur iFrame qui permettra l'affichage
En fait il faut "recalculer" toute l'url, y compris la session. La SYNAPP s'execute dans l'environnement du REDY mais n'a pas "conscience" des URL comme pour les synoptiques.

![image](https://github.com/user-attachments/assets/43005cf0-6592-4bfc-bc69-81e5f4031050)

```
SYNAPPS-STUDIO-ACTOR|{"type":"display/iframe","key":"iframe1","properties":{"verticalAlignment":"expand"},"additionalDefs":{"redyPath":{"type":"text","label":"Chemin REDY"}},"additionals":{"redyPath":"/easy/dashboard"},"bindings":{"properties.url":{"sourceType":"relative","sourcePath":"self","path":"additionals.redyPath","scripts":[{"path":"properties/url/binding/onReadTransform","value":"return  `${this.dataStores.redy.host.domain}/WSID${this.dataStores.redy.host.session.sid}${context.value}`;"}]}},"events":{"properties/url/binding/onReadTransform":["return  `${this.dataStores.redy.host.domain}/WSID${this.dataStores.redy.host.session.sid}${context.value}`;"]}}

**Comment by Baptiste-ALL** [comment]
Merci pour votre réponse Nicolas j'ai bien compris pour supprimé la barre de navigation à gauche en jouant sur la marge négative par contre je ne trouve pas l'endroit ou mettre le Nod pour qu'il l'affiche directement le second ou troisième etc. à la place du premier 
![image](https://github.com/user-attachments/assets/f94057e5-c96e-49b9-9823-69c5dd23a3dd)

car cela me met 
![image](https://github.com/user-attachments/assets/9bc5bc8b-f1b1-46bc-9b3b-9c0b5609570a)

**Comment by WITNicolas** [comment]
Il manque le "/" juste avant votre url
/easy/Dashboard-1?NodId=146539&

**Comment by Baptiste-ALL** [comment]
merci faute d'inattention
